### PR TITLE
Fixed Form showing

### PR DESCRIPTION
**Pull request by hustbeef**

### CODE REVIEW

File: mff/Form.bas [hunk 1]
```diff
@@ -1244,10 +1244,10 @@ Namespace My.Sys.Forms
 				If IsWindowVisible(FHandle) = 0 Then Exit While
 			Wend
 			Visible = False
-			For i = 0 To pApp->FormCount -1
-				If pApp->Forms[i]->Child > 0 Then Cast(Form Ptr, pApp->Forms[i]->Child)->Enabled = True
+			For i = 0 To pApp->FormCount - 1
+				If pApp->Forms[i] = Cast(Form Ptr, pApp->Forms[i]->Child) Then Cast(Form Ptr, pApp->Forms[i]->Child)->Enabled = True
```
[the remark this answers is not in the document]
Comment (hustbeef) [reply mff/Form.bas:1248]: Yes. you are right. the trouble is pApp->Forms[i]->Child = 0 sometimes

**Comment by hustbeef** [comment]
Yes. you are right. the trouble is pApp->Forms[i]->Child = 0 sometimes

**Comment by XusinboyBekchanov** [comment]
It has already been checked there: If pApp->Forms[i]->Child > 0 Then

File: mff/OpenFileControl.bas [hunk 2]
```diff
@@ -339,7 +339,7 @@ Namespace My.Sys.Forms
 			If This.Parent <> 0 AndAlso This.Parent->Handle <> 0 Then
 				ThreadID = ThreadCreate(@CreateWnd, @This)
 				Do While FHandle = 0
-					Sleep(200, 1)
+					Sleep(300, 1)
```
Comment (XusinboyBekchanov) [inline mff/OpenFileControl.bas:342]: Why is Sleep here? If FHandle = 0 will not exit the CreateWnd procedure.

**Pull request closed without merging**